include/ur_client_library/control/script_command_interface.h

```diff
@@ -147,6 +147,17 @@ class ScriptCommandInterface : public ReverseInterface
    */
   bool endToolContact();
 
+  /*!
+   * \brief Set friction compensation for the torque_command. If true the torque command will compensate for friction,
+   * if false it will not.
+   *
+   * \param friction_compesation Will set a friction_compensation variable in urscript, which will be used when calling
+   * torque_command
+   *
+   * \returns True, if the write was performed successfully, false otherwise.
+   */
+  bool setFrictionCompensation(const bool friction_compensation);
```

Comment:
I've seen many discussions around toggling  a feature on and off, in my opinion having a clear parameter name that leaves no ambiguities what a `true` value means is helpful in these situations.

```suggestion
   * \param friction_compensation_enabled Will set a friction_compensation variable in urscript, which will be used when calling
   * torque_command
   *
   * \returns True, if the write was performed successfully, false otherwise.
   */
  bool setFrictionCompensation(const bool friction_compensation_enabled);
```